fixed-requirements.txt

```diff
@@ -18,7 +18,7 @@ requests[security]==2.22.0
 apscheduler==3.6.3
 gitpython==2.1.11
 jsonschema==2.6.0
-pymongo==3.7.2
+pymongo==3.10.0
```

Comment:
As I remember `requirements.txt` needs to be generated too based on this

Reply:
Yep, need to add generated files, was waiting on tests to pass first :)